### DISCUSSION TITLE
Getting error "'Clickhouse' medatata.json already exists" while trying to create backup

### DISCUSSION THREAD

**Discussion by gavishdewangan** [Altinity/clickhouse-backup#1008] [Q&A] [answered]
I am trying to create remote backup via backup actions, I am getting this error 'Clickhouse' medatata.json already exists. Just want to know when do we encounter this error and what is the fix for it?

**Comment by Slach** [comment, marked as answer]
Actually, it means you already created backup with the same name `ClickHouse`

better use `POST /backup/actions` with request body
```json
{"command":"create_remote <uniq_backup_name>"}
```
and poll `GET /backup/status` to watch progress

add 
```yaml
general:
  keep_backup_local: -1  
```
to automatically remove local backup after successful upload
